openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditor.interface.ts

```diff
@@ -44,7 +44,7 @@ export type EditorType = 'markdown' | 'wysiwyg';
 
 export interface RichTextEditorProp extends HTMLAttributes<HTMLDivElement> {
   autofocus?: boolean;
-  initialValue: string;
+  initialValue?: string;
```

Comment:
Why we are making initialValue optional? if we are doing so then we should have a default value in the editor.

Reply:
To make it work with `Form.Item` and yes it has default value
<img width="704" alt="Screenshot 2023-08-25 at 4 36 57 PM" src="https://github.com/open-metadata/OpenMetadata/assets/12962843/3a35b5f5-068b-4d09-bc09-350786d69ad5">